CONTRIBUTING.rst

```diff
@@ -0,0 +1,39 @@
+contributing
+------------
+
+in order for a pull request to be merged, please make sure it meets the following criteria:
+
+1. all unit tests pass for all supported versions of python and django.
+2. every newly added line of code is exercised by at least one unit tests.
+3. all code is compliant with PEP8 style conventions.
+
+
+development setup
+-----------------
+to setup a development environment, run the following commands::
+
+  ~/django-queryset-csv/$ pip install -r dev_requirements.txt
+  ~/django-queryset-csv/$ python setup.py install
```

Comment:
`setup.py develop` is more handy when working on a package

https://pythonhosted.org/setuptools/setuptools.html#development-mode